### ISSUE TITLE
[Menu design][Accessibility] Tools submenu: VoiceOver does not announce menu item role

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23465]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the iOS Menu Redesign
- Found on Firefox Beta iOS app 134.0
- Open any webpage

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open the menu from the hamburger menu icon button at the bottom of the screen
3. As the action sheet appears on screen, navigate to the Tools submenu and double tap to activate
4. Swipe through the menu items and observe VoiceOver behavior

### Expected behavior / Actual behavior

As mentioned throughout the [Figma file](https://www.figma.com/design/RFz9fYtotQCQuinwcZujZt/%5B%5D-Menu-Redesign?node-id=19728-177454&t=ispcxT90kORu5xuz-4), there are menu items that should be announced by VoiceOver with the menu item role. They would require `role="menuitem"` (See [ARIA: menuitem role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitem_role) )

### Device & build information
* Device: iPhone 15, iOS 18.1.1
* Build version: Firefox Beta 134.0
* First seen version: First time I've seen this issue

### Notes
Attachments: 

https://github.com/user-attachments/assets/92e2e308-701a-4fd1-84cb-2980d711d7aa

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10746)

**Comment by data-sync-user** [comment]
➤ Natalie Stroud commented:

Canceling-  mixed up mobile with web Matt P
